Runtime/Frameworks/UIToolkit/VisualElements/SvgElement.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UIElements;
+using Object = UnityEngine.Object;
+
+#if REACT_VECTOR_GRAPHICS
+using Unity.VectorGraphics;
+#endif
+
+public class SvgElement : Image
+{
+    private static readonly string styleName = "TabButtonStyles";
+    private static readonly string UxmlName = "Svg";
+
+    private static Type _vectorImageUtilsType;
+    private static MethodInfo _makeVectorImageAsset;
+
+#if REACT_VECTOR_GRAPHICS
+#if !UNITY_WEBGL && !UNITY_IOS && !UNITY_IPHONE && !UNITY_WSA && !UNITY_WSA_10_0
+    private static MakeVectorDelegate _makeVectorHook;
+
+    /// <summary>
+    ///     Delegate of MakeVector method
+    /// </summary>
+    /// <remarks>This method is not supported on iOS and Web, and needs to fallback to standard reflection</remarks>
+    delegate void MakeVectorDelegate(List<VectorUtils.Geometry> geometry, uint gradientResolution, out Object asset,
+        out Texture2D texture2D);
+#endif
+#endif
+
+    [SerializeField]
+    private string _rawSvg;
```

Comment:
Should be `rawSvg` according to this project's naming conventions so far. We don't use `_` at the start of private members. Only camelCase.